repl/src/dotty/tools/repl/terminal/SpecialKeys.scala

```diff
@@ -32,12 +31,16 @@ object SpecialKeys {
   val Home  = Alt+"OH"
   val End   = Alt+"OF"
 
+  val Tab = 9.toChar.toString
```

Comment:
This is in the ammonite source, we'd like to keep it as close to upstream as possible :)